.pre-commit-config.yaml

```diff
@@ -0,0 +1,36 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.6.0
+    hooks:
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+
+  - repo: https://github.com/pycqa/flake8
```

Comment:
Can't you replace flake8 and its plugins by ruff alone with its rules

Reply:
Ah, yep

Reply:
I thought you would add some ruff settings to select the removed plugins' rules

Reply:
I'm fine with the defaults, honestly